cmake/app/boilerplate.cmake

```diff
@@ -556,8 +556,13 @@ if(CONFIG_QEMU_TARGET)
       Enable Qemu supported ethernet driver like e1000 at drivers/ethernet"
     )
   elseif(CONFIG_NET_QEMU_ETHERNET)
+    if(CONFIG_ETH_QEMU_EXTRA_ARGS)
+      set(NET_QEMU_ETH_EXTRA_ARGS ",${CONFIG_ETH_QEMU_EXTRA_ARGS}")
+    else()
+      set(NET_QEMU_ETH_EXTRA_ARGS "")
```

Comment:
Those two lines can be deleted.
If `${NET_QEMU_ETH_EXTRA_ARGS}` is not defined below when used, then it's simply empty.